### PR TITLE
Delete duplicated constant

### PR DESCRIPTION
**Pull request by nownabe**
Hi! ☀️ 

I deleted duplicated constant in VisionAPI FeatureTypes.

### CODE REVIEW

File: vision/google/cloud/vision/feature.py [hunk 1]
```diff
@@ -30,7 +30,6 @@ class FeatureTypes(object):
     LABEL_DETECTION = 'LABEL_DETECTION'
     TEXT_DETECTION = 'TEXT_DETECTION'
     SAFE_SEARCH_DETECTION = 'SAFE_SEARCH_DETECTION'
-    IMAGE_PROPERTIES = 'IMAGE_PROPERTIES'
```
Comment (dhermes) [inline vision/google/cloud/vision/feature.py]: Can you also move `LABEL_DETECTION` so it is in alphabetical order?

**Comment by nownabe** [comment]
ordered :+1:

**Comment by dhermes** [comment]
Thanks @nownabe!

This LGTM. @daspecster do we need these hard-coded values or can you lean on protobufs more? (Not part of this PR, but this PR clearly indicates that writing this by hand is problematic.)

**Comment by daspecster** [comment]
@dhermes the goal down the road is to really use the protobufs almost entirely. So that's definitely a good direction to head IMHO.

**Pull request merged**